### ISSUE TITLE
Picasso Library Not Loading Images in Android 5.0

### ISSUE THREAD

**Issue by lopst** [square/picasso#825]
I have a ListView extending ArrayAdapter. I am using a view holder patter for the getView and here is a part of my getView method which loads images using picasso.

So here is the problem: this works fine for OS < Android 5.0, but in the case of Android 5.0 (Lollipop), it looks like Picasso is fetching these images when my app is installed and run for the very first time, but when I launch the app again, the images don't load. Not at all sure what the problem is. I am not loading huge images, you can assume all the images that I am loading are only of the size of a small icon/thumbnail (around 120X120). I am using Picasso 2.4.0 for my application and the phone that I am using for testing is the Nexus 4.

**Comment by meisterfuu** [comment]
This has to be a special case in your project. Picasso works very well for me on Android 5.0 on my Nexus 4 and 7.

More information is needed to help you here.

**Comment by JakeWharton** [comment]
Agree. Please show code, logs, example URLs, etc. You haven't provided enough information here for us to know what you are doing.

**Comment by loody** [comment]
I have the same question,but it works ok blow Lollipop(5.0),this is my code :

 private void drawThumbPhoto(String url, final Canvas canvas, final float x, final float y, final Paint paint) {
        Picasso.with(mContext).load(url).transform(new CircleTransform()).resize(80, 80).into(new Target() {
            @Override
            public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
                try {
                    if (y < 0) return;
                    canvas.drawBitmap(bitmap, x, y, paint);
                } catch (Exception e) {
                    e.printStackTrace();
                }
            }

            @Override
            public void onBitmapFailed(Drawable errorDrawable) {

            }

            @Override
            public void onPrepareLoad(Drawable placeHolderDrawable) {

            }
        });
    }
